### PR TITLE
Set image-minimizer to specify image width ...

### PR DESCRIPTION
**Pull request by yashpalgoyal1304**
#### What is it?
- [ ] Bugfix (user facing)
- [ ] Feature (user facing)
- [ ] Codebase improvement (dev facing)
- [x] Meta improvement to the project (dev facing)

#### Description of the changes in your PR

- set the width of the image, instead of the height
see the rationale mentioned in the linked issue

#### Before/After Screenshots/Screen Record

- Before:

https://user-images.githubusercontent.com/19423063/204108377-dd920f3b-eadc-4d65-b567-84a8cda03270.mp4

- After:

https://user-images.githubusercontent.com/19423063/204108380-b94bb158-f5f8-43a2-b64b-bae868a8d111.mp4

#### Fixes the following issue(s)

- Fixes https://github.com/TeamNewPipe/NewPipe/issues/9469 for portrait like orientations

#### Testing

* Try adding the test images at https://github.com/yashpalgoyal1304/NewPipe/issues/1
* And for resizing the viewerport: use "Device Toolbar" in the browser's dev tools, 
* ... which in chromium is triggered by: in caret arrow notation](https://en.wikipedia.org/wiki/Caret_notation): `^↑i` > `^↑m` or say in [Emacs notation](https://www.emacswiki.org/emacs/EmacsKeyNotation): `C-I` > `C-M`

#### Due diligence
- [x] I read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by yashpalgoyal1304** [comment]
I have made the value to exactly result in current behaviour for the default available landscape width, however I do think there's scope of improvement there - as the white landscape image looks so gigantic compared with the full portrait screenshot.

File: .github/workflows/image-minimizer.js [hunk 1]
```diff
@@ -86,7 +88,7 @@ module.exports = async ({github, context}) => {
         if (shouldModify) {
             wasMatchModified = true;
             console.log(`Modifying match '${match}'`);
-            return `<img alt="${g1}" src="${g2}" height=${IMG_MAX_HEIGHT_PX} />`;
+            return `<img alt="${g1}" src="${g2}" width=${(IMG_MAX_HEIGHT_PX * probeAspectRatio).toFixed(0)} />`;
```
Comment (yashpalgoyal1304) [inline .github/workflows/image-minimizer.js:91]:
```suggestion
            return `<img alt="${g1}" src="${g2}" width=${Math.min(600, (IMG_MAX_HEIGHT_PX * probeAspectRatio).toFixed(0))} />`;
```

**Review by Stypox — APPROVED** [review]
Thanks! I was wondering whether there was a fix for this, too :-)

**Comment by yashpalgoyal1304** [comment]
hey, while testing for these changes in my own repo, i changed the actions settings - but now, i can't remember what the defaults were, so, can u please guide on what i should set these settings to:

image

**Comment by Stypox** [comment]
"Allow all actions", I think

**Comment by yashpalgoyal1304** [comment]
okay! :handshake:

**Pull request merged**